src/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_net_command.py

```diff
@@ -77,7 +77,10 @@ def __init__(self, cmd_id, seq, text, is_json=False):
             as_dict["pydevd_cmd_id"] = cmd_id
             as_dict["seq"] = seq
             self.as_dict = as_dict
-            text = json.dumps(as_dict)
+            try:
+                text = json.dumps(as_dict)
+            except TypeError:
+                text = json.dumps(as_dict, default=lambda o: str(o))
```

Comment:
```suggestion
                text = json.dumps(as_dict, default=str)
```
Wouldn't this achieve the same without the extra `lambda` function?